package.json

```diff
@@ -58,13 +58,11 @@
     "grunt-sails-linker": "~0.10.1",
     "grunt-sync": "~0.2.3",
     "i18n": "~0.5.0",
-    "include-all": "~0.1.6",
```

Comment:
It's used by the sails-new generator (it's copied into new projects).  Without it, Grunt fails on new projects.